src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/SimplifyBooleanReturnCheck.java

```diff
@@ -51,6 +51,41 @@
  * <pre>
  * &lt;module name=&quot;SimplifyBooleanReturn&quot;/&gt;
  * </pre>
+ * <p>Example: </p>
+ * <pre>
+ * public class Test {
+ *
+ *  private boolean cond;
+ *  private Foo a, b;
+ *
+ *  public boolean check1() {
```

Comment:
Please add one more example of violation (some other case, can be checked in test sources for this check)